tests/test-libraries/Makefile

```diff
@@ -191,6 +191,39 @@ $(eval $(call FatFrameworkTemplate,ios-fat,iphoneos,iphonesimulator))
 $(eval $(call FatFrameworkTemplate,tvos-fat,tvos,tvsimulator))
 $(eval $(call FatFrameworkTemplate,watchos-fat,watchos,watchsimulator))
 
+define XCTemplate
+.libs/by-platform/$(1)/XTest.framework/XTest: $(foreach arch,$(3),.libs/$(2)/libtest$(4).$(arch).dylib) | .libs/by-platform/$(1)/XTest.framework
```

Comment:
Any reason you can't use the existing `XTest.framework`s?

```
$ ls -lad tests/test-libraries/.libs/*/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/ios-fat/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/iphoneos/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/iphonesimulator/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/macos/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/tvos-fat/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/tvos/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/tvsimulator/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/watchos-fat/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/watchos/XTest.framework
drwxr-xr-x  4 rolf  wheel  128 Nov 24 08:20 tests/test-libraries/.libs/watchsimulator/XTest.framework
```

Reply:
That should work, it came from https://github.com/rolfbjarne/xamarin-macios/commit/xcframework#diff-c0db661233e8bd837a40c8de3bdbb802b63d5edb88fa262a82d9799ee39e50ebR155 , but then I renamed it XTest since XCTest is the Apple framework for Xcode unit tests and something (linking or codesigning) did not appreciate the renaming.

Reply:
Oh I see what happened here. My point was that this code is more complicated than it needs to be; in any case I fixed it and pushed the fix.

Reply:
thanks, even simpler of the changes I made locally